run_tests.py

```
@@ -0,0 +1,25 @@
+import unittest
+
+def run_tests():
+    # Start coverage tracking
+    cov = coverage.Coverage()
+    cov.start()
+
+    # Discover and run tests
+    loader = unittest.TestLoader()
+    tests = loader.discover('Tests')
+    runner = unittest.TextTestRunner()
+    result = runner.run(tests)
```

Ruff (0.8.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)